package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "dev": "next dev -p $CONNECT_ACCOUNT_PORT",
     "build": "next build",
-    "lint": "eslint **/*.json \"{src,pages,bin,doc,tests}/**/*.{js,ts,jsx,tsx}\"",
+    "lint": "next lint && eslint \"{src,pages,bin,doc,tests}/**/*.{js,ts,jsx,tsx}\"",
```

Comment:
We don't check json files anymore ?

Reply:
We used to not do it for a long time, and the `next` preset seems to break the `json` linting. I made a task for it.